src/vizier/services/adaptive_export/internal/clickhouse/apply_test.go

```
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package clickhouse
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"sync/atomic"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)